src/components/ApiRefRouting/ScrollRouter.js

```diff
@@ -18,12 +19,21 @@ let lastScrollPosition = 0;
 const routeMap = new Map();
 const elementMap = new Map();
 
-export const ScrollRouter = ({ children }) => {
+let contentDom;
+
+export const ScrollRouter = ({ children, initialActive = "" }) => {
   const initialLoadCheck = React.useRef(false);
-  const activeNodeRef = React.useRef();
+  const [activeNode, setActiveNode] = React.useState({
+    ref: null,
+    id: initialActive,
+  });
   const trackedElementsRef = React.useRef([]);
   const isScrollingDown = React.useRef(false);
 
+  React.useEffect(() => {
+    contentDom = document.querySelector(`#${DOM_TARGETS.contentColumn}`);
```

Comment:
This might be a premature optimization, but since this _can_ be selected once, I figured why not 🤷 Especially since my other changes here mean that the effect will be running more frequently.